generate.sh

```diff
@@ -42,11 +42,11 @@ DST_KEYS_TEST=keys_test.go
 
 # Cleanup.
 rm -rf ${DST_DOC} \
-	${DST_ERRORS} ${DST_ERRORS_TEST} \
-	${DST_FEATURES} ${DST_FEATURES_TEST} \
-	${DST_FLAGS} ${DST_FLAGS_TEST} \
-	${DST_TYPES} ${DST_TYPES_TEST} \
-	${DST_KEYS} ${DST_KEYS_TEST}
+  ${DST_ERRORS} ${DST_ERRORS_TEST} \
+  ${DST_FEATURES} ${DST_FEATURES_TEST} \
+  ${DST_FLAGS} ${DST_FLAGS_TEST} \
+  ${DST_TYPES} ${DST_TYPES_TEST} \
+  ${DST_KEYS} ${DST_KEYS_TEST}
 
```

Comment:
Please, use the same formatting for the whole file (`\t` or spaces).

Reply:
Updated.

Reply:
The formatting of the file still be mixed, as example:

tab:
https://github.com/tarantool/go-iproto/blob/c2bb250cb826aa09d9e494daae4ced5e487dbde8/generate.sh#L72

2 spaces:
https://github.com/tarantool/go-iproto/blob/c2bb250cb826aa09d9e494daae4ced5e487dbde8/generate.sh#L80

Reply:
Updated.